src/ServicePulse.Host/vue/src/components/monitoring/EndpointList.vue

```diff
@@ -30,23 +35,58 @@ onMounted(async () => {
     <!--Table headings-->
     <div class="table-head-row">
       <div class="table-first-col">
-        <endpoint-list-sortable-column>Endpoint name</endpoint-list-sortable-column>
+        <EndpointListSortableColumn :is-active="activeColumn === 'name'" :sort-by="'name'" @click="updateActiveColumn('name')">Endpoint name</EndpointListSortableColumn>
       </div>
       <div class="table-col">
-        <endpoint-list-sortable-column v-tooltip title="Queue length: The number of messages waiting to be processed in the input queue(s) of the endpoint.">Queue Length<template #unit>(MSGS)</template></endpoint-list-sortable-column>
+        <EndpointListSortableColumn
+          :is-active="activeColumn === 'queueLength'"
+          :sort-by="'queueLength'"
+          @click="updateActiveColumn('queueLength')"
```

Comment:
Needing to restate the column name 3 times could lead to subtle copy-paste issues. This might be addessed by the suggestion about using `v-model` below.

Reply:
I agree.  I wasn't happy with it either.  Changing this to a `v-model` worked a lot better.